tests/utils.rs

```diff
@@ -38,26 +35,31 @@ pub fn cargo_bin() -> Command {
         .unwrap_or_else(|| Command::cargo_bin("ouch").expect("Failed to find ouch executable"))
 }
 
-/// Run a command inside of another folder.
+/// Creates files in the specified directory.
 ///
-/// example: `run_in("/tmp", "touch", "a b c")`
-pub fn run_in(folder: impl AsRef<Path>, bin: impl AsRef<OsStr>, args: &str) -> io::Result<Output> {
-    Command::new(bin)
-        .args(args.split_whitespace())
-        .current_dir(folder)
-        .output()
-}
-
-// write random content to a file
+/// ## Example
+/// 
+/// ```no_run
+/// let (_dropper, dir) = testdir().unwrap();
+/// create_files_in(dir, &["file1.txt", "file2.txt"]);
+/// ```
+pub fn create_files_in(dir: &Path, files: &[&str]) {
+    for f in files {
+     std::fs::File::create(dir.join(f)).unwrap();
+    } 
+ }
+
+/// Write random content to a file
 pub fn write_random_content(file: &mut impl Write, rng: &mut impl RngCore) {
-    let mut data = Vec::new();
-    data.resize(rng.gen_range(0..4096), 0);
+    let mut data = vec![0; rng.gen_range(0..4096)];
```

Comment:
This is mentioned in the commit that changed this line but I'll add the [clippy lint](https://rust-lang.github.io/rust-clippy/master/index.html#slow_vector_initialization) in the PR for convenience.